app/javascript/components/components/Header.js

```diff
@@ -14,8 +16,9 @@ class Header extends Component {
     console.log("current_user:", current_user)
     return (
       <>
-        <h1>React in Rails with Devise</h1>
+        <h1>nest.</h1>
```

Comment:
Just a reminder that it is not a a good practice to have logs in your main branch.